thesis.tex

```diff
@@ -30,7 +30,7 @@
 \input{sections/other/_1_dedication}
 \input{sections/other/_2_acknowledgement}
 \input{sections/other/_3_contribution_of_authors}
-\setcounter{table}{-1}
+\setcounter{table}{0}
```

Comment:
can you add a comment here, mentioning this fixes https://github.com/bitslab/uic-cs-thesis-template/issues/3#issuecomment-2455616743 ?

with that, good to go and ill merge

Reply:
Just did, looks ok?

Reply:
looks great. I'm squashing and committing now.  UIC engineering and math grad students will owe you one for the rest of time now :)

Reply:
Thanks! Just glad to help a little! your template is what’s really saving us all! :D